cmd/eskip/mediadefaults.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
I kind of agree with dan here... Having a defaultWrite/defaultRead and assigning them to the commands is more readable to me. But I would mix in my comment https://github.com/zalando/skipper/pull/76/files#r48719213 here as well to get rid of the map. All those maps from `string->someFunc` are a smell in my nose.